### ISSUE TITLE
TQDM Extension

### ISSUE THREAD

**Issue by derks** [datafolklabs/cement#399]
Might be cool to build a simple extension to integrate progress bars.

 * https://pypi.python.org/pypi/tqdm

**Comment by akhilman** [comment]
I am using https://pypi.python.org/pypi/progressbar2 for now, also has idea to implement extension for cement but have no clue how cement integration can simplify already simple usage.

**Comment by derks** [comment]
Agreed... usage for progress bars is pretty simple for most cased, though there are other cases thight might make sense to have a helper for it.    For example, `click`'s implementation of it:

 * http://click.pocoo.org/5/utils/#showing-progress-bars

**Comment by akhilman** [comment]
Is it ok if interface will be almost like click's one but accessible through `app.progressbar`?

I would like to do this task. But not now.

**Comment by derks** [comment]
Yeah, I would probably approach it that way.
